core/domain/services/assets/ReactAssetManager.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace EventEspresso\core\domain\services\assets;
+
+use DomainException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\services\assets\AssetManager;
+use EventEspresso\core\services\collections\DuplicateCollectionIdentifierException;
+
+
+/**
+ * ReactAssetManager
+ * assets for routes that utilize React
+ *
+ * @package EventEspresso\core\domain\services\assets
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class ReactAssetManager extends AssetManager
+{
+    const JS_HANDLE_REACT_DEV = 'react-development';
+    const JS_HANDLE_REACT_PROD = 'react-production';
+    const JS_HANDLE_REACT_DOM_DEV = 'react-dom-development';
+    const JS_HANDLE_REACT_DOM_PROD = 'react-dom-production';
+    // base URLs
+    const URL_BASE_UMD_REACT = 'https://unpkg.com/react@16/umd/';
+    const URL_BASE_UMD_REACT_DOM = 'https://unpkg.com/react-dom@16/umd/';
```

Comment:
What if we want to work offline sometimes? May be bundle it instead?

Reply:
I thought the idea was to use the CDN?

but ya, I can copy those scripts to the vendor folder